Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/FilteredDPE.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "FilteredDPE.h"
+#include "AzToolsFramework/UI/DocumentPropertyEditor/ui_FilteredDPE.h"
+
+namespace AzToolsFramework
+{
+    FilteredDPE::FilteredDPE(QWidget* parentWidget)
+        : QWidget(parentWidget)
+        , m_filterAdapter(AZStd::make_shared<AZ::DocumentPropertyEditor::ValueStringFilter>())
+        , m_ui(new Ui::FilteredDPE)
```

Comment:
nit: explicitly call constructor in new statement (there's some very specific edge cases in which it's not the same, so let's keep it consistent :D )
```
, m_ui(new Ui::FilteredDPE())
```

Also, could/should this be `aznew`?

Reply:
fixed, thanks!